SECURITY.md

```diff
@@ -2,24 +2,26 @@
 
 ## Supported Versions
 
-We provide security updates for the following releases.
+We provide security updates for `develop` and for the last two
+"major" `0.x` release series of Spack. Security updates will be
```

Comment:
Shouldn't this be minor, not major? You could even remove the 0 from the version number.

Reply:
See https://github.com/spack/spack/releases/tag/v0.20.0:

> v0.20.0 is a major feature release.

For all intents and purposes, they're major, b/c we're still at 0.x and we've kind of shifted things over one.

What if it said "`develop` and the last two stable releases"?  We mention that these are stable in the release docs.

Reply:
This:

> "develop and the last two stable releases"

sounds slightly better, but I think also the current version would be good (we always used "major" when releasing 0.x versions)